lib/cmd/fh3/credentials.js

```diff
@@ -22,20 +22,16 @@ function credentials(argv, cb) {
   var action = args[0];
   if ("list" === action) {
     return listCredentials(args, cb);
-  } /*else if ("create" === action) {
-    return createCredentials(args, cb);
-  } else if ("update" === action) {
-    return updateCredentials(args, cb);
-  } else if ("delete" === action) {
-    return deleteCredentials(args, cb);
-  }*/ else {
+  } else {
```

Comment:
It should be useful as you have CRUD operations related to credentials. But this code need to be tested before rather then removed